blackjack.js

```diff
@@ -0,0 +1,117 @@
+document.addEventListener("DOMContentLoaded",() => {
+    let deck_id;
+    let playerOne = document.querySelector("#playerOne")
+    let house = document.querySelector("#house")
+    let scorePlayer = 0;
+    let computerScore = 0
+    let winner;
+
+const fetchCards = async ()=> {
+    try{
+        let res = await axios.get("https://deckofcardsapi.com/api/deck/new/");
+        deck_id = res.data.deck_id
+        let shuffled = await axios.get(`https://deckofcardsapi.com/api/deck/${deck_id}/shuffle/`)
+        shuffled
+    } catch (err) {
+        console.log("error")
+        
+    }
+}
+
+const drawCards  = async (deck_id,div,num,score)=>{
+    let Num1 = 0
+    try {
+        let drawData = await axios.get(`https://deckofcardsapi.com/api/deck/${deck_id}/draw/?count=${num}`)
+      
+        for(let i = 0; i < num; i++){
+            let image = document.createElement("img")
+            let url = drawData.data.cards[i].image
+            let value = drawData.data.cards[i].value
+            Num1 += getValue(value,score)
+            image.src = url
+    
+            div.appendChild(image) 
+            
+        }
+       
+    } catch(err) {
+        console.log(err);
+    }
+  
+    score += Num1
+    let p = document.createElement("p")
```

Comment:
You might want to create this outside this scope, so any clicks after they first want append/update your previous p tag